handler/src/main/java/io/netty/handler/ssl/JdkAlpnApplicationProtocolNegotiator.java

```diff
@@ -141,7 +141,9 @@ public SSLEngine wrapSslEngine(SSLEngine engine, ByteBufAllocator alloc,
                 return isServer ? JettyAlpnSslEngine.newServerEngine(engine, applicationNegotiator)
                         : JettyAlpnSslEngine.newClientEngine(engine, applicationNegotiator);
             }
-            throw new RuntimeException("Unable to wrap SSLEngine of type " + engine.getClass().getName());
+            throw new UnsupportedOperationException("ALPN not supported (used java version: "
+                    + PlatformDependent.javaVersion() + ", min java version: 9). Unable to wrap SSLEngine of type '"
```

Comment:
This seems to not be true anymore. Starting `1.8.0u252`, ALPN is supported. 
(see https://www.eclipse.org/jetty/documentation/current/alpn-chapter.html#alpn-versions)

Reply:
oh interesting... Didn't know this was backported.

Reply:
So, the `jdkAlpnSupported` method should be updated to handle this new "feature".

Reply:
@cescoffier working on a PR as we speak

Reply:
@cescoffier https://github.com/netty/netty/pull/10196